docs/csharp/local-functions-vs-lambdas.md

```diff
@@ -15,44 +15,50 @@ ms.assetid: 368d1752-3659-489a-97b4-f15d87e49ae3
 
 ### Local functions compared to Lambda expressions
 
-In some use cases, you could create a lambda expression and use it
-without needing to create a local function. Here's an example async
-method that does just that:
+At first glance, local functions and lambda expressions are very similar.
+Depending on your needs, local functions may be a much better and simpler
+solution.
 
-[!code-csharp[TaskLambdaExample](../../samples/snippets/csharp/new-in-7/AsyncWork.cs#36_TaskLambdaExample "Task returning method with lambda expression")]
```

Comment:
Since this is the only place that references `36_TaskLambdaExample`, and it's being deleted, should that `#region` be deleted too?